deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -91,7 +99,9 @@ def collect_screenshots_for_languages(path)
 
         UI.important("Framed screenshots are detected! 🖼 Non-framed screenshot files may be skipped. 🏃") if prefer_framed
 
-        language = File.basename(lng_folder)
+        language_dir_name = File.basename(lng_folder)
+        language = available_languages[language_dir_name.downcase] || language_dir_name
```

Comment:
Do we know that the absence of a language from that endpoint's list means the language isn't supported by Apple? I think I like the idea of falling back to previous behavior in this case (and not blocking the user), but there is an argument to be made for logging a warning here, maybe?